packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/CreateRulesTests.java

```diff
@@ -215,7 +216,7 @@ public void siteConsumerCannotCreateRule()
         restClient.assertLastError().containsSummary("Insufficient permissions to manage rules");
     }
 
-    /** Check that a siteManager can create a rule in a private folder. */
+    /** Check that a siteManager can create a rule in a folder in a private site. */
     public void siteManagerCanCreateRule()
```

Comment:
Do we need `@Test (groups = { TestGroup.REST_API, TestGroup.RULES })` annotation for the method to be executed?